Makefile.am

```diff
@@ -26,6 +26,12 @@ EXTRA_DIST = \
 	subprojects/libffi.wrap \
 	subprojects/proxy-libintl.wrap \
 	subprojects/zlib.wrap \
+	subprojects/google-benchmark.wrap \
+	perf/meson.build \
+	perf/perf-draw.hh \
+	perf/perf-extents.hh \
+	perf/perf-shaping.hh \
+	perf/perf.cc \
```

Comment:
not happy on putting perf on dist, as there it is hard to check we in autotools dist in meson, yet, isn't that but undecided.

Reply:
so may is ready to merge if you see the change in the right direction.